src/EnergyPlus/WaterCoils.cc

```diff
@@ -980,6 +983,11 @@ namespace WaterCoils {
 			MyCoilReportFlag = true;
 			InitWaterCoilOneTimeFlag = false;
 			PlantLoopScanFlag = true;
+
```

Comment:
water coils, and HXAssistedWaterCoils, now get controller name and index. Can't do this in GetIn;put for water coils since this will cause a recursive call to GetInput.